.travis.yml

```diff
@@ -356,9 +356,13 @@ py27_linux_build_engine: &py27_linux_build_engine
   name: "Build Linux native engine and pants.pex (Py2.7 PEX)"
   env:
     - docker_image_name=travis_ci
-    # We also build fs_util.
-    - docker_run_command="./build-support/bin/ci.sh -2b
-                          && ./build-support/bin/release.sh -f"
+    # Note that we also build fs_util, to take advantage of the rust code built during bootstrapping.
```

Comment:
We don't want this full comment because we now explain this all here:

https://github.com/pantsbuild/pants/blob/36b2ace51415ba605339bdb59b624b5e040ad0df/.travis.yml#L296-L300